### ISSUE TITLE
Not working with python.exe. Library searches for python3.exe

### ISSUE THREAD

**Issue by jeraldpraveen** [extremeheat/JSPyBridge#151]
Python is installed in c:\Python\Python311. I am able to fetch python version with command 'python --version'.
But the command 'python3 --version' does not fetch python version. However, pythonia expects python3.exe

Since python3.exe is not found, pythonia throws the below error.
`BridgeException: Attempt to access 'fileImport' failed. Python didn't respond in time (100000ms), look above for any Python errors. If no errors, the API call hung.`

<img width="691" alt="image" src="https://github.com/user-attachments/assets/ab09863f-b73a-4c29-b824-d72462bb1f67">

**Comment by extremeheat** [comment]
Yes, we should probably move away from `python3` to `python` default. But as mentioned you can currently fix this by setting the environment variable ahead of time to the correct path of your Python bin.
